modules/core/src/lib/model.js

```diff
@@ -236,25 +240,34 @@ export default class Model {
     this.updateModuleSettings(moduleSettings);
     this.setUniforms(uniforms);
 
-    const logPriority = this._logDrawCallStart(2);
+    let logPriority;
+
+    if (this.debug) {
```

Comment:
why not use existing `log.priority` variable, and change this if check to `log.priority >= 2` , instead of adding new `this.debug` variable.

Reply:
It's currently just logging, but I want to move towards all debug logic being separated out from the main render logic. This mirrors what I've done in `Program`.